src/commands/functions/create.js

```diff
@@ -216,7 +218,7 @@ async function downloadFromURL(flags, args, functionsDir) {
   }
 
   try {
-    fs.mkdirSync(fnFolder, { recursive: true })
+    mkdirRecursiveSync(fnFolder)
```

Comment:
Since this function is already marked `async` should we use the asynchronous version of `mkdir`?

Reply:
I kept using sync in there to keep it consistent with the other `fs.*` calls.

Reply:
Let me know if you want me to change these fs calls to async or just keep it the way it is. @erezrokah 

I don't think this is a big issue tho, probably can be done in another PR.

Reply:
No worries you can keep it `sync`